e2e/nextjs/pages/_error.tsx

```diff
@@ -0,0 +1,20 @@
+// pages/_error.tsx
+import NextError from 'next/error'
```

Comment:
This new `pages/_error.tsx` file reports Page Router SSR errors from the client. It's more reliable than attempting to send them from the server.